frontend/src/app/organization/organization-offers-list/organization-offers-list.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container" *ngIf="offers.length; else noOffers;">
+  <table class="table table-striped mt-3">
+    <tr>
+      <th></th>
+      <th>Tytuł</th>
+      <th>Miejsce</th>
+      <th>Czas obowiązywania</th>
+      <th>Status</th>
+      <th></th>
+    </tr>
+    <tr *ngFor="let offer of offers">
+      <td>
+        <a class="volontulo-link-circle" [routerLink]="['/offers', offer.slug, offer.id]"
+            [style.backgroundImage]="'url(' + offer.image +')'">
+        </a>
+        <!--<a class="crop-circle" [routerLink]="['/offers', offer.slug, offer.id]">-->
```

Comment:
One more thing that I missed - if not needed anymore and everything is working as expected then please remove those commented tags :)

Reply:
I forgot about it. Sorry.